corehq/preindex/tasks.py

```diff
@@ -2,9 +2,16 @@
 from celery.task.base import periodic_task
 
 from corehq.preindex.accessors import index_design_doc, get_preindex_designs
+from corehq.util.decorators import serial_task
+from django.conf import settings
 
 
-@periodic_task(run_every=crontab(minute=0), queue='background_queue')
+@periodic_task(run_every=crontab(minute='*/5'), queue=settings.CELERY_PERIODIC_QUEUE)
+def run_continuous_indexing_task():
+    preindex_couch_views.delay()
+
+
+@serial_task('couch-continuous-indexing', timeout=60 * 60, queue=settings.CELERY_PERIODIC_QUEUE, max_retries=0)
```

Comment:
yeah, using serial_task with max_retries=0 makes it so that we never have two of these tasks running at the same time, and if a task is spawned while another is running, it will be ignored (within the timeout window, so within a 1 hour period in this case).

we don't have a serial_periodic_task util yet, so that's the only reason i had to split them into two tasks.